pkg/dns/service.go

```diff
@@ -184,7 +202,7 @@ func (s *Service) SetEndpoints(ctx context.Context, addresses []gatewayv1beta1.G
 			Targets:       []string{ep},
 			RecordType:    "A",
 			SetIdentifier: ep,
-			RecordTTL:     60,
+			RecordTTL:     v1alpha1.TTL(dnsPolicy.Spec.TTL.Value),
```

Comment:
This is assuming a policy will always set a TTL. Is that really what we want?

Reply:
decided to remove TTL from policy